lib/_tls_wrap.js

```diff
@@ -860,6 +865,20 @@ TLSSocket.prototype.renegotiate = function(options, callback) {
   return true;
 };
 
+TLSSocket.prototype.exportKeyingMaterial = function(length, label, context) {
+  validateUint32(length, 'length', true);
+  validateString(label, 'label');
+  if (context) {
```

Comment:
```suggestion
  if (context !== undefined) {
```

Reply:
Also, non-blocking style nit... if blocks that contain a single line do not need the `{` `}` brackets :-)